src/select-project.html

```diff
@@ -0,0 +1,72 @@
+
+<link rel="import" href="shared-styles.html">
+<link rel="import" href="../bower_components/vaadin-combo-box/vaadin-combo-box.html">
+<link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
+
+
+<dom-module id="select-project">
+  <template>
+    <style>
+    </style>
+
+    <!-- Fetch an array of users to be shown in the grid -->
+    <iron-ajax id="projects" url="/dummy/projects.json" handle-as="json" last-response="{{projects}}"></iron-ajax>
+
+    <vaadin-combo-box filtered-items="[[items]]" label="Project" item-label-path="title" on-filter-changed="_projectsFilter" selected-item="{{selectedProject}}">
+      <template>
+        [[item.title]]
+      </template>
+    </vaadin-combo-box>
+
+  </template>
+
+  <script>
+    class SelectProject extends Polymer.Element {
+      static get is() { return 'select-project'; }
+
+      static get properties () {
+        return {
+          selectedProject: {
+            type: Object,
+            notify: true
+          },
+          items: {
+            type: Array,
+            value: []
+          },
+          projects: {
+            type: Array,
+            value: []
+          }
+        }
+      }
+
+      constructor () {
```

Comment:
No need to call constructor or any other lifecycle functions if there's no overriding functionality – `super()` will be then called automatically.